Separate GUILD_CREATE fields from Guild object (#4888)
hammerandchisel/discord-api-docs: docs/resources/Guild.md

###### Guild Structure

> info

> > Previously, fields specific to the [GUILD_CREATE](#DOCS_TOPICS_GATEWAY/guild-create) event were documented here. They have now been moved there.

Reviewer: ```suggestion
> Fields specific to the `GUILD_CREATE` event are listed in the [Gateway documentation](#DOCS_TOPICS_GATEWAY/guild-create).
```

Author: Ah, thank you, I like that much more.
